app/users/serializers/AdminSerializer.py

```diff
@@ -103,17 +104,34 @@ def to_representation(self, instance):
         Customize the representation of an Admin instance.
 
         This method customizes the serialized representation of an Admin instance,
-          including handling superuser rights.
+        including handling superuser rights.
 
         Args:
             instance (Admin): The admin instance to represent.
 
         Returns:
             dict: The customized serialized representation of the admin instance.
         """
+        CODENAME_TO_RIGHTS = {
```

Comment:
CODENAME_TO_RIGHTS Instead of computing it inside the function every time, we can move it out if it's used frequently